### PR TITLE
Feature/stub demo

### PR DESCRIPTION
**Pull request by grimadas**
Fancy pipeline log for iroha demo

### CODE REVIEW

File: irohad/ametsuchi/ametsuchi_stub.cpp [hunk 1]
```diff
@@ -18,19 +18,25 @@
 #include <ametsuchi/ametsuchi_stub.hpp>
 #include <ametsuchi/mutable_storage_stub.hpp>
 #include <ametsuchi/temporary_wsv_stub.hpp>
+#include <iostream>
 
 namespace iroha {
   namespace ametsuchi {
 
     std::unique_ptr<TemporaryWsv> AmetsuchiStub::createTemporaryWsv() {
+      std::cout << "[\033[33mAmetsuchi\033[0m] creating temporary wsv" << std::endl;
```
Comment (satellitex) [inline irohad/ametsuchi/ametsuchi_stub.cpp]: Aer they formal logs?
If they are formal, they should be used by `logger`.
(ref : https://github.com/hyperledger/iroha/blob/develop/test/module/irohad/logger/logger_test.cpp )

Comment (grimadas) [inline irohad/ametsuchi/ametsuchi_stub.cpp]: No, this is just for demonstration, we will change it after to logger. Can we assign to logger the color of the text ?

Comment (satellitex) [inline irohad/ametsuchi/ametsuchi_stub.cpp]: Yes, longer can use the color of text same how.

Comment (satellitex) [inline irohad/ametsuchi/ametsuchi_stub.cpp]: Yes, `logger` can use the color of text.

File: irohad/main/iroha-main.cpp [hunk 2]
```diff
@@ -66,13 +66,18 @@ int main(int argc, char *argv[]) {
   iroha::torii::ToriiStub torii(tp, qp);
   // shows required order of execution, since callbacks are called synchronously
   peer_communication_service.subscribe_on_proposal();
+
+  // ***** test starts here *****
```
Comment (l4l) [inline irohad/main/iroha-main.cpp]: maybe separate to related test file&function then?

Comment (MizukiSonoko) [inline irohad/main/iroha-main.cpp]: Or Can you at least use option (`$./iroha-main -t`)

**Comment by MizukiSonoko** [comment]
Really we need to merge this commit?

**Comment by grimadas** [comment]
Thanks everyone, we don't need to merge this. I'll close this PR

**Pull request closed without merging**